Added a note on QUIC multiplexing (#104)
Haivision/srt-rfc: draft-sharabayko-srt-over-quic.md

and connection migration. It was designed to become a faster alternative to the TCP protocol {{RFC7323}}.

An Unreliable Datagram Extension to QUIC {{QUIC-DATAGRAM}} adds support for sending and receiving

> unreliable datagrams over a QUIC connection, but transits the responsibility of multiplexing different kinds
> of datagrams, or flows of datagrams to an application protocol.

Reviewer: ```suggestion
unreliable datagrams over a QUIC connection, but transfers the responsibility for multiplexing different kinds
of datagrams, or flows of datagrams, to an application protocol.
```

SRT {{SRTRFC}} is a UDP-based transport protocol. Essentially, it can operate over any unreliable datagram transport.

> SRT provides loss recovery, stream multiplexing mechanisms.
> In live streaming configuration SRT provides an end-to-end latency-aware mechanisms for packet loss recovery.

Reviewer: ```suggestion
SRT provides loss recovery and stream multiplexing mechanisms.
In its live streaming configuration SRT provides an end-to-end latency-aware mechanism for packet loss recovery.
```

SRT employs a listener (server) / caller (client) model. The data flow is bi-directional and 
independent of the connection initiation - either the sender or receiver can operate 
as listener or caller to initiate a connection.

The SRT protocol provides an internal multiplexing mechanism, allowing multiple SRT connections
to share the same UDP port, providing access control functionality to identify the caller on the listener side.

> It is exactly what QUIC DATAGRAM wants to be a responsibility of the application protocol.

Reviewer: ```suggestion
This mechanism is exactly what QUIC DATAGRAM describes as a responsibility of the application protocol.
```
